### PR TITLE
Add boolean to uninstall for docker

### PR DESCRIPTION
**Pull request by michaelgugino**
By default, we won't attempt to unstall docker
or it's configuration files, including storage.

Users can opt-in to docker uninstall via
openshift_uninstall_docker=True, which defaults
to false.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1569553

### CODE REVIEW

File: playbooks/adhoc/uninstall_openshift.yml [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+    with_subelements:
+    - "{{ files.results | default([]) }}"
+    - files
+
+  - file: path={{ item.1.path }} state=absent
+    with_subelements:
+    - "{{ directories.results | default([]) }}"
+    - files
+
+  - shell: systemctl daemon-reload
```
[the remark this answers is not in the document]
Comment (michaelgugino) [reply playbooks/adhoc/uninstall_openshift.yml]: Most of this is copy/paste from what we had.  It needs a lot of cleanup generally, but I'm focused on minimum changes for not destroying docker for now.

[the remark this answers is not in the document]
Comment (vrutkovs) [reply playbooks/adhoc/uninstall_openshift.yml]: Agree, this can be cleaned up later

**Comment by michaelgugino** [comment]
bot, retest this please

**Comment by michaelgugino** [comment]
@sdodson @vrutkovs I have tested this locally without running the docker uninstall bits, it works well against 3.11 and 3.9; I didn't test 3.10 specifically, but it should be pretty much the same as 3.11.

**Comment by vrutkovs** [comment]
/lgtm

Worke fine on RHEL or Atomic

**Comment by vrutkovs** [comment]
bot, retest this please

**Pull request merged**